scale/storage/brokers/host/host_broker.py

```diff
@@ -0,0 +1,99 @@
+"""Defines a broker that mounts a local host directory into the task container as its backend storage"""
```

Comment:
As we grow the number of brokers, I figured it would be useful to have their own directories for additional files/code they might need. The NFS broker not in its own directory is the old one that will be deleted once we create the new one (in its own directory).